resources/nats-streaming/templates/pre-install-hook.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Namespace }}-ns-check
```

Comment:
why not .Release.Name??  (for all of them)

Reply:
Done